CRM/Report/Form/Contribute/Summary.php

```diff
@@ -58,6 +58,7 @@ class CRM_Report_Form_Contribute_Summary extends CRM_Report_Form {
    * Class constructor.
    */
   public function __construct() {
+    $batches = CRM_Batch_BAO_Batch::getBatches();
```

Comment:
This isn't a very efficient way to `SELECT count(id)`... is your expectation that there are never many records in this table so it's ok?

Reply:
@colemanw my expectation is that this was already being done - just further down - so it's no worse

Reply:
Ah, ok but then the query was running twice. Fixed it by switching the lower call to use your new variable.
Also removed a redundant item from the array which PHPStorm helpfully flagged.